src/styles/isomer-cms/elements/buttons.scss

```diff
@@ -91,8 +91,4 @@ button {
       box-shadow: unset;
     }
   }
-
-  &:last-child {
-    margin-right: 0px;
-  }
```

Comment:
Actually, is it possible to do this outside of the file? We've been avoiding modification of these files because they tend to have a bunch of unexpected changes haha

Reply:
Hmm I was struggling quite a bit with the CSS specificity because it seems like the `last-child` selector has a higher specificity than the `marginRight` prop in the Button component. I also did a cursory look at other parts of the CMS and it doesn't seem like any component was affected.

Reply:
tbh, we might wanna err on the side of caution here (avoiding visual breakage) but fwiw, this is increasingly become legacy so we might wanna prune too...

i'm fine with both either way but if deciding to stay safe, passing `styles` directly could work to override specificity?

Reply:
I'm inclined to remove this as I think by default, our buttons have the `margin-right` property set to 0px. Any usages of the `marginRight` or `mr` props in our codebase do not apply to the last child button element, so I think it would be safe to remove this.